dandiapi/api/tests/test_users.py

```diff
@@ -162,38 +128,33 @@ def test_user_search_multiple_matches(api_client, user, user_factory, social_acc
         'john_doe',
         'john_foo',
     ]
-    users = [user_factory(username=username) for username in usernames]
-    social_accounts = [social_account_factory(user=user) for user in users]
+    users = [UserFactory.create(username=username) for username in usernames]
 
     assert api_client.get('/api/users/search/?', {'username': 'odysseus'}).data == [
-        serialize_social_account(social_account) for social_account in social_accounts[:3]
+        serialize_user(user) for user in users[:3]
     ]
 
 
 @pytest.mark.django_db
-def test_user_search_limit_enforced(api_client, user, user_factory, social_account_factory):
+def test_user_search_limit_enforced(api_client):
+    user = UserFactory.create()
     api_client.force_authenticate(user=user)
 
     usernames = [f'odysseus_{i:02}' for i in range(20)]
-    users = [user_factory(username=username) for username in usernames]
-    social_accounts = [social_account_factory(user=user) for user in users]
+    users = [UserFactory.create(username=username) for username in usernames]
 
     assert api_client.get('/api/users/search/?', {'username': 'odysseus'}).json() == [
-        serialize_social_account(social_account) for social_account in social_accounts[:10]
+        serialize_user(user) for user in users[:10]
     ]
 
 
 @pytest.mark.django_db
-def test_user_search_extra_data(api_client, user, social_account, social_account_factory):
+def test_user_search_extra_data(api_client):
     """Test that searched keyword isn't caught by a different field in `extra_data`."""
+    user = UserFactory.create(social_account__extra_data__test='odysseus')
```

Comment:
Note this syntax. This is why it's nice when we can use `factory.Dict` for properties, as it can be easily overridden selectively.